### ISSUE TITLE
I have a question about model training.

### ISSUE THREAD

**Issue by jp1924** [mkunes/w2v2_audioFrameClassification#3]
There was a problem setting up the environment for model verification of the paper.

I don't know what hyperparameter I should use to conduct the verification because the paper lacks information related to the experimental environment.

If it is not a problem, I would like to know the hyperparameter used to print the results of Table 1, 2, and 3 of the paper.

Hyperparameters that I want to know mainly include
- learning rate
- learning rate scheduler
- warmup step
- epoch
- batch_size
- weight_decay
- fp16

**Comment by mkunes** [comment]
The number of epochs was 5 in all cases, as mentioned in the paper (admittedly it's a bit buried in the text).

Anything that's not explicitly mentioned used the default values from the script. If I remember correctly, I originally copied many of the defaults from some examples in the Transformers documentation, and we never bothered to change them. (So they're not optimized or anything; there might be better configurations.)

This is the exact call used for the multitask results on AMI, in Table 3 (it says `--num_epochs 7`, but we used the 5th epoch for our results, since the extra two didn't help on the development set):

    python3 /storage/plzen4-ntis/home/mkunes/transformers/wav2vec2_audioFrameClassification_multitask.py \
        --model_checkpoint facebook/wav2vec2-base \
        --feature_extractor_name facebook/wav2vec2-base \
        --file_train /storage/plzen4-ntis/projects/speaker_recog/AMI/Wav2vec2Transformer_multitask_OSD-VAD-SCD/AMI-train_multitask_fuzzy_OSD-VAD-SCD_22-09-22.json \
        --finetuning_framework pytorch \
        --num_epochs 7 \
        --batch_size 1 \
        --model_save_dir /storage/plzen4-ntis/projects/speaker_recog/AMI/Wav2vec2Transformer_multitask_OSD-VAD-SCD/AMITrain_fuzzy0.2_22-09-22_20s10s_7epochs_noSigmoid_noWeights_seed9999 \
        --max_duration 20 \
        --mode train \
        --epochs_between_checkpoints 1 \
        --dataset_caching 1 \
        --seed 9999 \
        --multitask

and for prediction on the test set (some of these arguments are unnecessary, but I copied it here exactly as we used it):

    epoch=5
    python3 /storage/plzen4-ntis/home/mkunes/transformers/wav2vec2_audioFrameClassification_multitask.py \
        --model_checkpoint /storage/plzen4-ntis/projects/speaker_recog/AMI/Wav2vec2Transformer_multitask_OSD-VAD-SCD/AMITrain_fuzzy0.2_22-09-22_20s10s_7epochs_noSigmoid_noWeights_seed9999/epoch${epoch} \
        --feature_extractor_name facebook/wav2vec2-base \
        --file_eval /storage/plzen4-ntis/projects/speaker_recog/AMI/Wav2vec2Transformer_multitask_OSD-VAD-SCD/AMI-test_multitask_fuzzy_OSD-VAD-SCD_22-09-22.json \
        --batch_size 1 \
        --file_output /storage/plzen4-ntis/projects/speaker_recog/AMI/Wav2vec2Transformer_multitask_OSD-VAD-SCD/AMITrain_fuzzy0.2_22-09-22_20s10s_7epochs_noSigmoid_noWeights_seed9999/epoch${epoch}/output_AMI.txt \
        --max_duration 20 \
        --mode eval \
        --epochs_between_checkpoints 1 \
        --dataset_caching 1 \
        --multitask

**Comment by jp1924** [comment]
Thank you for your answer @mkunes!
